server/lib/isbn/isbn.js

```diff
@@ -14,6 +15,8 @@ export const normalizeIsbn = text => {
 
 export const isNormalizedIsbn = text => /^(97(8|9))?\d{9}(\d|X)$/.test(text)
 
+export const findIsbns = text => text.match(Isbn)
```

Comment:
The `Isbn` regex misses the `g` flag, so it can not work: you get a single matching result array instead of a an array of matches. Ex:
```js
"9782203202283 9782749307015".match(/(97(8|9))?[\d-]{9,14}([\dX])/g)
// => [ '9782203202283', '9782749307015' ]
"9782203202283 9782749307015".match(/(97(8|9))?[\d-]{9,14}([\dX])/)
// => [ '9782203202283', '978', '8', '3', index: 0, input: '9782203202283 9782749307015', groups: undefined ]
```

Then, you can not have the `g` flagged regex as a shared regex in `lib/regex`, as you would risk entering [one of the darkest corners of the JS RegExp implementation](https://stackoverflow.com/questions/1520800/why-does-a-regexp-with-global-flag-give-wrong-results) (see 7e21472773f76e730cd56405d05124ffa77ebd01), thus my suggestion to generate a regex object at every function call.